File: spologp_v/lista2sala-C/questao9.c
#include<stdio.h>

int main()
{
    int v1, v2;
    printf("Digite o valor 1:");
    scanf("%i", &v1);
    printf("Digite o valor 2:");
    scanf("%i", &v2);

if((v1%4==0) || (v2%4==0)){
    printf("Esses valores são divisíveis por 4: %i, %i, %i\n", v1, v2);
}
else{
    printf("\nEsses valores não são divisíveis por 4\n");
}

if((v1%5==0) || (v2%5==0)){
    printf("\nEsses valores são divisíveis por 5: %i, %i, %i\n", v1, v2);
}
else{
    printf("\nEsses valores não são divisíveis por 5\n");
}

return 0;
}

File: spologp_v/lista2sala-python/questao9.py
v1 = int(input("Digite o primeiro valor:"))
v2 = int(input("Digite o segundo valor:"))

if(v1%4==0) and (v2%4==0):
    print("Esses valores são divisíveis por 4:\n", v1, v2)
else:
    print("Esses valores não são divisíveis por 4\n")

if(v1%5==0) and (v2%5==0):
    print("Esses valores são divisíveis por 5:", v1, v2)
else:
    print("Esses valores não são divisíveis por 5")